pkg/scheduler/metrics/metric_recorder.go

```diff
@@ -99,26 +99,39 @@ type MetricAsyncRecorder struct {
 
 	// stopCh is used to stop the goroutine which periodically flushes metrics.
 	stopCh <-chan struct{}
-	// IsStoppedCh indicates whether the goroutine is stopped. It's used in tests only to make sure
+	// isStoppedCh indicates whether the goroutine is stopped. It's used in tests only to make sure
 	// the metric flushing goroutine is stopped so that tests can collect metrics for verification.
-	IsStoppedCh chan struct{}
+	isStoppedCh chan struct{}
```

Comment:
All `IsStoppedCh` is replaced with `WaitUntilStopped`. No longer need to make it a public field.